### PR TITLE
Rsik's Fluff Item

### PR DESCRIPTION
**Pull request by Agameofscones**
Fluff PR for Rsik, he told me the concept was pre-approved, so don't yell at me yell at him.

ckey: Xydonus
Character Name: Rsik Ugsharki
Job: It's a modkit? Idk ask him

alttext
image
:cl: Triiodine
add: Rsik's fluff katana ~~modkit~~
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
I wonder why he just didn't get a katana item rather than a modkit

**Comment by KasparoVy** [comment]
Modkits are all the rage

**Comment by taukausanake** [comment]
I mean if it's just a toy katana. The worse you can do with it is commit ~~sudoku~~ seppuku. Do we even have any real katana weapons on the server anywhere?

**Comment by Agameofscones** [comment]
It's just a loadout possible fluff item now, defined exactly the same way as the toy katana with a special suicide emote. At Rsik's request.

**Comment by KasparoVy** [comment]
Only way to get katanas are buss or opening crates gathered mining.

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -135,6 +135,27 @@
...
+	sharp = 0
+	flags = CONDUCT
+	slot_flags = SLOT_BELT | SLOT_BELT
+	throwforce = 5
+	w_class = WEIGHT_CLASS_NORMAL
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+
+/obj/item/fluff/rsik_katana/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] tries to stab [src] into [user.p_their()] stomach! Except [src] shatters! [user.p_they(TRUE)] look[user.p_s()] as if [user.p_they()] might die from the shame.</span>")
```
Comment (taukausanake) [inline code/modules/customitems/item_defines.dm]: So the blade shatters but it doesn't break? Not to be nitpicky but I'd imagine it to delete itself or have a broken sprite afterwards

**Pull request merged**